### ISSUE TITLE
TopBar Color / Menu background Color are the same

### ISSUE THREAD

**Issue by PowershellScripter** [d2phap/ImageGlass#1118]
### System information:
 - Windows OS version: 
 - ImageGlass version: 

### To Reproduce
1.  Changed Topbar color to black
2.  Went into Settings > Toolbar
3. 
4. 

### Actual behavior:
Turned Menu Background Black

### Expected behavior:
Only topbar color should change

### Screenshots:
![image](https://user-images.githubusercontent.com/71091773/127535374-c0cc510e-5fcb-40b1-9c9a-194f6031c503.png)

**Comment by fire-eggs** [comment]
You've made the assumption that "topbarcolor" in the theme refers to the _title_ bar. In fact, it refers to the _toolbar_.

Nothing has changed since you submitted issue #929: changing the title bar color is not supported.

**Comment by d2phap** [comment]
Toolbar, Viewer and Thumbnail bar background colors are defined in theme pack.
You can manually edit the theme

**Comment by PowershellScripter** [comment]
> You've made the assumption that "topbarcolor" in the theme refers to the _title_ bar. In fact, it refers to the _toolbar_.
> 
> Nothing has changed since you submitted issue #929: changing the title bar color is not supported.

I was not talking about the titlebar, I was in fact talking about the toolbar. I changed it to black and it changed the ribbon menu background color in the settings menu as well.



> Toolbar, Viewer and Thumbnail bar background colors are defined in theme pack.
> You can manually edit the theme

Yes, I know I can edit the theme, ive been using ImageGlass for quite some time now. My concern is that you set the ribbon menu (the part in the settings that allows you to add or remove items from the toolbar) color to the same ID as the toolbar. This means when I change the toolbar color to black, it also changes the background of the ribbon menu to black as I had posted a screenshot of. The problem I am having is that because I change the toolbar background color to black and it changes the ribbon menu background color to black as well, the font is also black so the text is difficult to read. Is there a way to change JUST THE TOOLBAR background color, without it changing the ribbon menu background color? Or a way to chnage the ribbon menu text color?
